Tests/StarknetTests/Accounts/AccountTest.swift

```diff
@@ -114,13 +114,13 @@ final class AccountTests: XCTestCase {
 
         let call = StarknetCall(contractAddress: ethContractAddress, entrypoint: starknetSelector(from: "transfer"), calldata: calldata)
 
-        let nonce = try await account.getNonce()
-        let feeEstimate = try await account.estimateFeeV1(call: call, nonce: nonce)
+        let nonce = try await account.getNonce().send()
+        let feeEstimate = try await account.estimateFeeV1(call: call, nonce: nonce).send()[0]
```

Comment:
The [change of estimate fee return type](https://github.com/software-mansion/starknet.swift/pull/187/files#diff-27821160cc3ae3a86a78970f05cff3a582fb6894068fe88eef74e693981c9effL157-R157) resulted in having to access element `[0]` every time we use this metod. Unfortunately there isn't any other way, because first we need to send the request and the access its result.